fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1300,8 +1301,8 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 			// simulation
 			TEST(true); // Semi-committed pipeline limited by MVCC window
 			//TraceEvent("ProxyWaitingForCommitted", pProxyCommitData->dbgid).detail("CommittedVersion", pProxyCommitData->committedVersion.get()).detail("NeedToCommit", commitVersion);
-			waitVersionSpan = Span(
-			    deterministicRandom()->randomUniqueID(), "MP:overMaxReadTransactionLifeVersions"_loc, { span.context });
+			// TODO - ljoswiak parent or link?
```

Comment:
Hmm this one is a bit debatable, I think it's fine to leave it as parent, but if you feel strongly it should be link instead I won't argue.

Reply:
Leaving as parent and removing TODO